web/src/main/java/no/nav/modiapersonoversikt/rest/api/Mapper.kt

```diff
@@ -88,6 +98,59 @@ private fun meldingMapping(melding: Melding): MeldingDTO =
         )
     )
 
+private fun henvendelseDTOMapping(henvendelse: SfHenvendelseDTO): TraadDTO {
+    val journalpost: JournalpostDTO? = henvendelse.journalposter?.firstOrNull()
+    val meldinger: List<MeldingDTO> = (henvendelse.meldinger ?: emptyList()).map { melding ->
+        MeldingDTO(
+            mapOf(
+                "id" to "${henvendelse.kjedeId}-${(henvendelse.hashCode())}",
+                "oppgaveId" to null,
+                "meldingstype" to meldingstypeFraSfTyper(henvendelse, melding),
+                "temagruppe" to henvendelse.gjeldendeTemagruppe,
+                "skrevetAvTekst" to melding.fra.ident, // TODO trenger remapping for hente ut navn
```

Comment:
Skal ta en nærmere titt på hvordan vi kan gjøre denne mapping fra ident til navn. Liker ikke helt denne to-delingen som PRen har nå.